README.md

```diff
@@ -1,7 +1,94 @@
-## Ion Hash Python
+# Amazon Ion Hash Python
+An implementation of [Amazon Ion Hash](http://amzn.github.io/ion-hash) in Python.
 
-A Python implementation of the Ion Hash Specification
+[![Build Status](https://travis-ci.org/amzn/ion-hash-python.svg?branch=master)](https://travis-ci.org/amzn/ion-hash-python)
+
+This package is designed to work with **Python 3.4+**.
+
+## Getting Started
+
+The following Python code calculates a hash for the Ion value `[1, 2, 3]` using the MD5 hash function:
+
+```python
+from io import BytesIO
+
+from amazon.ion.core import IonEventType
+from amazon.ion.reader import blocking_reader
+from amazon.ion.reader import NEXT_EVENT
+from amazon.ion.reader_managed import managed_reader
+from amazon.ion.reader_text import text_reader
+
+from amazon.ionhash.hasher import hash_reader
+from amazon.ionhash.hasher import hashlib_hash_function_provider
+from amazon.ionhash.hasher import HashEvent
+
+
+ion = b'[1, 2, 3]'
+hash_function = "md5"
+
+reader = hash_reader(
+    blocking_reader(managed_reader(text_reader(), None), BytesIO(ion)),
```

Comment:
This is making me realize we should probably provide a utility in ion-python that helps users get the right kind of coroutine for their data (`text_reader` or `binary_reader`). Probably exposing the first few lines of [`simpleion.load`](https://github.com/amzn/ion-python/blob/master/amazon/ion/simpleion.py#L329).

Reply:
yes.  a simple `to_ion_hash(algorithm)` function would likely be helpful, too